authentication/views.py

```diff
@@ -1,9 +1,51 @@
-from django.shortcuts import render
-from django.http import HttpResponse
-from django.core import serializers
+import json
+from django.http import JsonResponse
 from authentication.models import *
+from django.contrib.auth import authenticate, login, logout
+from django.views import View
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+import re
 
-# Create your views here.
-def returnTodoJson(request):
-    data = Todo.objects.all()
-    return HttpResponse(serializers.serialize("json", data), content_type="application/json")
+@method_decorator(csrf_exempt, name='dispatch')
+class RegisterView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        username = data['username']
+        password = data['password']
+        email = data['email']
+        if AppUser.objects.filter(username = username).exists() or AppUser.objects.filter(email = email).exists():
```

Comment:
use django rest framework since we are making this an api, 
https://www.django-rest-framework.org/

Reply:
we can utilize serializers for validation and sending the response

Reply:
ok, will fix it asap.